config/authorities/raster_formats.yml

```diff
@@ -0,0 +1,5 @@
+terms:
+  - id: image/tiff; gdal-format=GTiff
+    term: GeoTIFF
+  - id: text/plain; gdal-format=AIGrid
+    term: AIGrid
```

Comment:
Should be: 
  - id: text/plain; gdal-format=AAIGrid
  - term: Arc/Info ASCII Grid

We should also add: 
- id: application/octet-stream; gdal-format=AIG
- term: Arc/Info Binary Grid